### PR TITLE
PoY v0.75

### PR DESCRIPTION
**Pull request by HrikB**
As the next phase of our iterative rollout of Proof of Yield, we want to fill out this section of the second page.
image

Also snuck in some other minor changes
- Altered rETH logo
- Fixed rounding inconsistency among this site and the dapp
- Ending pagination beyond the 26th of April

### CODE REVIEW

**Review by micahalcorn — CHANGES_REQUESTED** [review]
- [x] Kill the blue icons
- [x] Repair pagination on first page so that it adds to the bottom instead of replacing the existing dates
- [ ] "Back to list" works but the back button breaks
- [ ] Detail page flickers white before showing content
- [ ] Are these dates actually UTC or based on when Keeper runs?
- [x] Reset scroll to top when clicking detail page

File: pages/proof-of-yield/[timestamp].tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  props: YieldOnDayProps;
+}> => {
+  const { timestamp } = context.query;
+
+  let timestampNumber: number | undefined;
+  let dailyStat: DailyStat;
+  if (
+    timestamp &&
+    typeof timestamp === "string" &&
+    (timestampNumber = parseInt(timestamp))
```
Comment (smitch88) [inline pages/proof-of-yield/[timestamp].tsx:68]: This feels like a bit of an odd syntax to put into the conditional. Is there a particular reason to set the value within it? 

If not, ideally we move this out into the next line within the brackets.

File: sections/proof-of-yield/DailyYield.tsx [hunk 2]
```diff
@@ -36,6 +41,10 @@ const DailyYield = ({ dailyStats }: DailyYieldProps) => {
     setStats(() => [...moreStats]);
   };
 
+  const validDate = (date: string) => {
+    return moment(date).isAfter("2023-04-26");
```
Comment (smitch88) [inline sections/proof-of-yield/DailyYield.tsx:45]: What is this related too? Exclusion of outlier data? Perhaps we really should be doing this within the endpoint or source of truth?

File: sections/proof-of-yield/DailyYield.tsx [hunk 3]
```diff
@@ -83,76 +92,97 @@ const DailyYield = ({ dailyStats }: DailyYieldProps) => {
             </TableHead>
             {width >= smSize && (
               <TableHead
-                info={`The sum of all assets currently held by OETH`}
-                className="px-4 md:px-8"
+                info={`Yield is distributed to all regular Ethereum wallets and any smart contracts or mult-sigs that have opted-in.`}
```
Comment (smitch88) [inline sections/proof-of-yield/DailyYield.tsx:95]: `multi` -> `multi`?

File: sections/proof-of-yield/DailyYield.tsx [hunk 4]
```diff
@@ -83,76 +92,97 @@ const DailyYield = ({ dailyStats }: DailyYieldProps) => {
...
-                )}
-                {/* {width >= smSize && (
-                <TableData className="pr-0 xl:pr-8">
-                  Ξ {commifyToDecimalPlaces(parseFloat(item.backing_supply), 2)}
-                </TableData>
-              )} */}
-                {/* <TableData className="px-6" align="center">
-                <ChartDetailsButton
-                  onClick={() => routeToYieldOnDay(item.date)}
+          {console.log(stats.filter((s) => validDate(s.date)))}
```
Comment (smitch88) [inline sections/proof-of-yield/DailyYield.tsx:108]: remove console log

File: sections/proof-of-yield/DayBasicData.tsx [hunk 5]
```diff
@@ -18,40 +17,33 @@ import { smSize, lgSize, xlSize } from "../../constants";
...
+const DayBasicData = ({
+  timestamp,
+  dailyStat,
+  sectionOverrideCss,
+}: DayBasicDataProps) => {
   const router = useRouter();
   const width = useViewWidth();
 
+  const yieldBonusProps: YieldBoostMultiplierProps = {
+    rawApy: parseFloat(dailyStat.raw_apy),
```
Comment (smitch88) [inline sections/proof-of-yield/DayBasicData.tsx:40]: Seems like `dailyStat` could be undefined without timestamp parsing. Either create a guard to not render this in the parent usage or to be safe just add in `?` w/ a default value, perhaps `0`

File: sections/proof-of-yield/DayBasicData.tsx [hunk 6]
```diff
@@ -176,17 +201,18 @@ const DayBasicData = ({ timestamp, sectionOverrideCss }: DayBasicDataProps) => {
                 </tr>
               </thead>
               <tbody>
-                {mockData.map((item) => (
+                {dailyStat.rebase_events.map((item) => (
```
Comment (smitch88) [inline sections/proof-of-yield/DayBasicData.tsx:204]: Should get use to `?` just to safeguard against any client errors throwing on `undefined`

File: utils/api/fetchDailyStats.ts [hunk 7]
```diff
@@ -6,13 +6,15 @@ interface DailyStatsResponse {
...
+  startAt?: number
 ): Promise<DailyStat[]> {
   try {
-    const endpoint = `${process.env.NEXT_PUBLIC_ANALYTICS_ENDPOINT}/api/v2/oeth/daily_stats/${end}`;
-    if (begin) endpoint.concat(`/${begin}`);
+    let endpoint = `${process.env.NEXT_PUBLIC_ANALYTICS_ENDPOINT}/api/v2/oeth/daily_stats/${end}`;
+    if (startAt) endpoint += `/${startAt}`;
     const response = await fetch(endpoint);
 
+    console.log(`Fetching daily stats from ${endpoint}`);
```
Comment (smitch88) [inline utils/api/fetchDailyStats.ts:16]: Console log

File: sections/proof-of-yield/DailyYield.tsx (hunk 2 above)
Comment (HrikB) [reply sections/proof-of-yield/DailyYield.tsx:45]: The endpoint returns from the beginning of contract launch (the 4/18) but we want it to begin after the first non-zero yield day (4/26)

**Review by smitch88 — APPROVED** [review]

**Review by smitch88 — APPROVED** [review]

**Review by micahalcorn — CHANGES_REQUESTED** [review]
As discussed in Discord, let's just change UTC to PT and add a helper icon with a tooltip. Then I'm good with deploying this and making some other tweaks as we go.

**Review by micahalcorn — APPROVED** [review]
